aperturedb/Operations.py

```diff
@@ -31,3 +35,39 @@ def rotate(self, angle, resize=False):
         }
 
         self.operations_arr.append(op)
+        return self
+
+    def flip(self, code: str) -> Operations:
+
+        op = {
+            "type": "flip",
+            "code": code,
+        }
+
+        self.operations_arr.append(op)
+        return self
+
+    def crop(self, x: int, y: int, width: int, height: int) -> Operations:
+
+        op = {
+            "type": "crop",
+            "x": x,
+            "y": y,
+            "width": width,
+            "height": height,
+        }
+
+        self.operations_arr.append(op)
+        return self
+
+    def interval(self, start: int, stop: int, step: int) -> Operations:
```

Comment:
this operation is only available for videos or clips, not sure how this will impact.

Reply:
I am guessing server will send back an appropriate response if a invalid combo land up.